docs/examples/ex36.py

```diff
@@ -0,0 +1,221 @@
+r"""Incompressible Hyperelasticity
+This example solves the governing equations describing the mechanical 
```

Comment:
Can you also add a short version of the description to `docs/listofexamples.rst` together with the image?

The image can be uploaded to this issue https://github.com/kinnala/scikit-fem/issues/431 after which you can get an url with right click.